content/TrueCommand/TCGettingStarted/AdminAccounts.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Creating the Admin Account"
+description: "This article describes how to create the administrator account in TrueCommand."
+weight: 30
```

Comment:
This needs to be weighted to appear immediately after the install articles. This one covers what the users sees first when attempting to log in and the interface overview covers what the user sees after logging in for the first time.

Reply:
This article, and the user accounts article, now appear immediately after the install articles.